OrbitGl/CallStackDataView.cpp

```diff
@@ -107,7 +107,7 @@ void CallStackDataView::OnDataChanged() {
 //-----------------------------------------------------------------------------
 Function* CallStackDataView::GetFunction(int a_Row) {
   int index = m_Indices[a_Row];
-  if (m_CallStack != nullptr && index < m_CallStack->m_Depth &&
+  if (m_CallStack != nullptr && index < static_cast<int>(m_CallStack->m_Depth) &&
```

Comment:
Is this a safe conversion? m_Depth is uint32_t and casting index to uint32_t would yield the same value, if we can be sure that index is >= 0. Casting m_Depth to int may overflow (though unlikely given the semantic of the value). 

Also, should we CHECK(index >= 0) and CHECK(a_Raw >= 0) here?

Reply:
Good point. I tried to make the least intrusive change and make the smallest assumption possible. So I assumed `m_CallStack->m_Depth` fits in a signed int. Your checks are also
very good, but adding these here is another rabbit hole. I saw opportunities for checks in all the locations in OrbitGl and OrbitCore.